ecosystem/ffmpeg_plugin/build_ffmpeg_plugin.sh

```diff
@@ -1,43 +1,87 @@
 #!/bin/bash
 
 # SPDX-License-Identifier: BSD-3-Clause
-# Copyright 2024 Intel Corporation
+# SPDX-FileCopyrightText: Copyright (c) 2024 Intel Corporation
 
 set -e
 
-if [ -n "$1" ]; then
-	ffmpeg_ver=$1
-else
-	# default to latest 7.0
-	ffmpeg_ver=7.0
-fi
+# Default values
+ffmpeg_ver="7.0"
+enable_gpu=false
+script_path="$(dirname $(readlink -f $0))"
+
+# Help message function
+usage() {
+    echo "Usage: $0 [options]"
+    echo "Options:"
+    echo "  -v <version>    Specify the FFmpeg version to build (default is 7.0)"
+    echo "  -g              Enable GPU direct mode during compilation"
+    echo "  -h              Display this help and exit"
+}
+
+# Parse command-line options
+while getopts ":v:hg" opt; do
+    case "${opt}" in
+        v)
+            ffmpeg_ver=${OPTARG}
+            ;;
+        g)
+            enable_gpu=true
+            ;;
+        h)
+            usage
+            exit 0
+            ;;
+        h)
```

Comment:
why do we have double "h)"? (Here and 4 line above)

Reply:
Yes missed it 
fixed in [d815791](https://github.com/OpenVisualCloud/Media-Transport-Library/commit/d815791a6643e3c587c432a98d984a86345168ef)